### PR TITLE
release: v1.16.2

### PR DESCRIPTION
**Pull request by RomainMuller**
Upgrades `jsii` in order to gain the code generation fix for incorrect
processing of structs passed as dictionary literals through a nested
lifted keyword argument (struct-in-struct).

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [1.16.2](https://github.com/aws/aws-cdk/compare/v1.16.1...v1.16.2) (2019-11-12)
```
Comment (shivlaks) [inline CHANGELOG.md]: heads up that this needs to be ##

Comment (RomainMuller) [reply CHANGELOG.md]: Ugh yeah- why's this not automatic though?

File: build.sh [hunk 2]
```diff
@@ -60,6 +60,8 @@ echo "==========================================================================
 echo "building..."
 time lerna run $bail --stream $runtarget || fail
 
-/bin/bash scripts/check-api-compatibility.sh
+if [ "${runtarget:-}" != "build" ]; then
+    /bin/bash scripts/check-api-compatibility.sh
+fi
```
Comment (eladb) [inline build.sh]: What's that?

File: bump.sh [hunk 3]
```diff
@@ -23,9 +23,6 @@ export NODE_OPTIONS="--max-old-space-size=4096 ${NODE_OPTIONS:-}"
 
 npx lerna version ${version} --yes --exact --force-publish=* --no-git-tag-version --no-push
 
-# Another round of install to fix package-lock.jsons
-/bin/bash ./install.sh
-
```
Comment (eladb) [inline bump.sh]: why?

File: package.json [hunk 4]
```diff
@@ -13,21 +13,8 @@
...
-    "jest": "^24.9.0",
-    "jsii-diff": "^0.20.3",
-    "jsii-pacmak": "^0.20.3",
     "lerna": "^3.18.4",
-    "nodeunit": "^0.11.3",
-    "nyc": "^14.1.1",
-    "standard-version": "^7.0.0",
-    "ts-jest": "^24.1.0",
-    "tslint": "^5.20.1",
-    "typescript": "~3.6.4"
```
Comment (eladb) [inline package.json]: ha?

File: build.sh (hunk 2 above)
Comment (RomainMuller) [reply build.sh]: If we're going to skip tests, we should skip *all* tests. This made the bump script take ~45 minutes though... only to then get a ~45 minutes PR validation... then a ~45 minutes build... Followed by a ~45 minutes documentation site generation... Who has this kind of time?

File: bump.sh (hunk 3 above)
Comment (RomainMuller) [reply bump.sh]: There are no `package-lock.json` files to fix anymore, and this step wasn't adding any value - just making things noisier and slower.

File: package.json (hunk 4 above)
Comment (RomainMuller) [reply package.json]: Dropped LOTS of un-needed dependencies (made install slower, introduced risk of taking "shadow" dependencies on those, etc...)

**Comment by RomainMuller** [comment]
The `jsii` bump is in it's own commit in the PR... (https://github.com/aws/aws-cdk/pull/4968/commits/76662640a8e9f113e4997b1a981410b1718da1c1)

**Pull request merged**